src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ErrorNotificationHandler.java

```diff
@@ -59,6 +70,10 @@ private void processMessage(ErrorMsg message) {
         service.processServiceBusMessage(message);
     }
 
+    private void logCompletion(String messageId) {
+        log.debug("Error notification consumed. ID {}", messageId);
```

Comment:
I wonder if this should not be info instead. It's useful information and if it's debug, we'll never see it, except for local env maybe.

Reply:
it would be duplicating log. service already announces the completion:
```
log.info("Error notification published. ID: {}", response.getNotificationId());
```

then it tries to save the db record. if in any case it fails - atm message will be dead lettered and relevant logs will be error'ed

Reply:
I think notification failures must be only dead lettered if they are bad request ?